### PR TITLE
Update kernel.c

### PR DESCRIPTION
**Pull request by andyLaurito92**
Adding extremely necessary documentation

### CODE REVIEW

**Review by scherman — APPROVED** [review]
LGTM. Good catch!

**Review by tbert12 — APPROVED** [review]

**Review by juanmanuelbouvier — APPROVED** [review]
LGTM

File: src/kernel.c [hunk 1]
```diff
@@ -34,7 +34,7 @@ kmain()
     _mascaraPIC2(0xEF); //Habilito mouse
 
 	_Sti();
-
+/* Method that initialites the shell */
```
Comment (juanmanuelbouvier) [inline src/kernel.c:37]: typo: initializes

**Comment by jorexe** [comment]
Thank you guys, I don't know how I couldn't see that mistake

**Pull request merged**